### PR TITLE
Remove flatten and map.values apoc procedures

### PR DESCRIPTION
**Pull request by angrykoala**
# Description
This PR removes `apoc.coll.flatten` from the constants list and removes `apoc.map.values` in favour of `[k in keys(value) | value[k]][0]`

### CODE REVIEW

**Comment by neo4j-team-graphql** [comment]
Made some changes to the GraphQL Toolbox? Or made some changes to the core library perhaps?

Check out your changes at
 http://neo4j-graphql-toolbox-stage-2222.surge.sh (=> for local development; use for instance [Neo4j Desktop](https://neo4j.com/download/))
or
 https://neo4j-graphql-toolbox-stage-2222.surge.sh (=> use to connect to an [Aura Free](https://neo4j.com/cloud/aura-free/) dbms)

File: packages/graphql/src/translate/translate-top-level-cypher.ts [hunk 1]
```diff
@@ -172,8 +172,8 @@ export function translateTopLevelCypher({
     } else {
         cypherStrs.push(`
             CALL apoc.cypher.doIt("${statement}", ${apocParamsStr}) YIELD value
-            WITH apoc.map.values(value, [keys(value)[0]])[0] AS this
-        `);
+            WITH [k in keys(value) | value[k]][0] as this
```
Comment (Liam-Doodson) [inline packages/graphql/src/translate/translate-top-level-cypher.ts:175]: Not blocking but is the convention not for `as` to be upper case?

**Review by Liam-Doodson — DISMISSED** [review]
Looks good to me other than the non-blocking comment!

File: packages/graphql/src/translate/translate-top-level-cypher.ts (hunk 1 above)
Comment (angrykoala) [reply packages/graphql/src/translate/translate-top-level-cypher.ts:175]: Good catch
```suggestion
            WITH [k in keys(value) | value[k]][0] AS this
```

**Pull request merged**